app/boards/shields/romac/romac.keymap

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020 Pete Johanson, Richard Jones
```

Comment:
@bmcgavin you good here? We've been checking with contributors before replacing these.

Reply:
Oops, Will revert if there is a concern. Just changed when i tried to change mine.

Reply:
No problem here.